File: ApiSample/DAL/Models/Depot.cs
using System;
using System.Collections.Generic;

namespace DAL.Models;

public partial class Depot
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public string Location { get; set; } = null!;

    public int OwenerId { get; set; }

    public virtual ICollection<Operation> Operations { get; } = new List<Operation>();

    public virtual Entity Owener { get; set; } = null!;
}

File: frontend_sample/types/depot.ts
import { ContainerInDepot } from "./container"

export type Depot = {
    id: number,
    name: string,
    location: string,
    owener: string
}

export type DepotExtraInfo = {
    containers: ContainerInDepot[]
}

export type DepotFullInfo = Depot & DepotExtraInfo;
